packages/node/src/Session.ts

```diff
@@ -100,6 +101,21 @@ const storageUtility = new StorageUtilityNode(defaultStorage, defaultStorage);
  */
 export const issuerConfigFetcher = new IssuerConfigFetcher(storageUtility);
 
+/**
+ * @hidden
+ */
+function isValidUrl(url: string): boolean {
```

Comment:
I look forward to dropping Node 18 support and being able to use https://developer.mozilla.org/en-US/docs/Web/API/URL/canParse_static